balancer.go

```diff
@@ -80,33 +83,42 @@ func NewBalancer(config *Config) (*Balancer, error) {
 		if errPing := sqlConn.Ping(); errPing != nil {
 			node.conn = sqlConn
```

Comment:
suggestion: since we always assing sqlConn to node.conn, we should do it right after setting sqlConn definitions.

Reply:
i can do it later, and support also config for `ConnMaxIdleTime`

Reply:
Will do in a follow up PR